src/AppInstallerCommonCore/Public/AppInstallerTelemetry.h

```diff
@@ -115,20 +127,22 @@ namespace AppInstaller::Logging
             std::string_view arpLanguage) const noexcept;
 
     protected:
-        TelemetryTraceLogger();
+
+        // Check for valid Json string and return a valid Json
+        std::string GetTelemetryCorelationJson() const;
 
         bool IsTelemetryEnabled() const noexcept;
 
         bool m_isSettingEnabled = true;
         std::atomic_bool m_isRuntimeEnabled{ true };
+        GUID m_activityId = GUID_NULL;
+        std::string m_telemetryCorelationJson = "{}";
+        std::string m_comCaller = "";
     };
 
     // Helper to make the call sites look clean.
     TelemetryTraceLogger& Telemetry();
 
-    // Turns on wil failure telemetry and logging.
-    void EnableWilFailureTelemetry();

```

Comment:
Will revert the location of this method's declaration to reflect no change to this method!